third_party/blink/renderer/core/animation/animation.h

```diff
@@ -316,6 +316,11 @@ class CORE_EXPORT Animation : public EventTargetWithInlineData,
   bool AnimationHasNoEffect() const { return animation_has_no_effect_; }
   bool AtScrollTimelineBoundary();
 
+  // RUN-1716
+  int RecordReplayId() const override {
+    return ScriptWrappable::RecordReplayId();
+  }
```

Comment:
This change seems unnecessary and also unrelated to the rest of the patch?

Reply:
It is necessary to resolve ambiguity: `Animation` has both `ContextLifecycleObserver` and `ScriptWrappable` as base class. I'll add a comment.